src/sentry/ingest/consumer/processors.py

```diff
@@ -272,10 +274,15 @@ def process_event(
                 )
 
         # remember for an 1 hour that we saved this event (deduplication protection)
-        cache.set(deduplication_key, "", CACHE_TIMEOUT)
+        with sentry_sdk.start_span(op="cache.set") as span:
+            span.set_data("deduplication_key", deduplication_key)
```

Comment:
curious what this one is for.. do we actually need that?

Reply:
No, it was a "just-in-case" for debugging but yeah probably not useful. Removed it.